packages/react-dom/src/__tests__/ReactDOMFizzServer-test.js

```diff
@@ -9,6 +9,8 @@
 
 'use strict';
 
+global.TextEncoder = require('util').TextEncoder;
```

Comment:
This is probably a good indication that this isn’t backwards compatible enough and that instead the config should import this from utils itself.